packages/web-react/src/components/Grid/Grid.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Grid from './Grid';
+import { SpiritGridProps } from '../../types';
+
+const gridVariants = [1, 2, 3, 4, 6, 12];
+
+export default {
+  title: 'Components/Grid',
+  argTypes: {
+    children: {
+      control: 'text',
+    },
+    mobile: gridVariants,
+    tablet: gridVariants,
+    desktop: gridVariants,
+  },
+};
+
+const Template = (args: SpiritGridProps) => <Grid {...args} />;
+
+export const ExampleGrid = Template.bind({});
+ExampleGrid.args = {
+  children: 'Grid',
+  mobile: 1,
+  tablet: 2,
+  desktop: 3,
```

Comment:
do you think this scale (mobile, table, desktop) will be enough? how do we want to scale this case?

Reply:
I think this scale is based on products needs and it is based on design/UX knowledge. I think that we can still go up to large or extra large devices.